app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -305,5 +323,43 @@ private boolean noDuplicateNumber() {
         }
         return noDuplicate;
     }
+
+    /**
+     * Set focus to the next EditText after picking contact
+     */
+    public void setFocusOnNextView() {
+
+        int index = -1;
+        View currentFocus = selectedButton;
+        String tag = (String)currentFocus.getTag();
+        try
+        {
+            index = Integer.parseInt(tag);
+            Log.d("index",String.valueOf(index));
+        }catch (NumberFormatException e)
+        {
+            e.printStackTrace();
+        }
+
+        if(index >= 0 && index < comradeEditText.size())
+        {
+            EditText nextFocus = comradeEditText.get(index);
+            if(nextFocus.getText().toString().equals(""))
+                nextFocus.requestFocus();
+            else
+                okButton.requestFocus();
+        }
+        else
```

Comment:
EditText 1 : filled
EditText 2 : filled
EditText 3 : unfilled
EditText 4 : filled
EditText 5 : filled
EditText 6 : FOCUS
After filling EditText 6, control goes to EditText 3